contracts/LoanManager.sol

```diff
@@ -72,12 +73,26 @@ contract LoanManager is Restricted, TokenReceiver {
         rates = _rates;
     }
 
+    /* @Deprecated: For compatibility with previous versions (without the margin parameter), e.g. old SB scripts */
```

Comment:
Why do we keep this? No client code uses it only local migration scripts so we could just remove

Reply:
All SB scripts would have to be modified (not only local, but rinkeby and mainnet too). In general we are trying to avoid modifying old SB scripts, which were already run.
Also all loan related unit tests would have to be modified without this.

Reply:
Ouch, I see. But I don't like the idea to carry fx forever because of SB scripts. Also compile time will increase as the number of SB scripts are growing. What if we archive old SB scripts to a folder which doesn't compile? 
(tests shouldn't be a reason to keep a deprecated fx :) I'm happy to do the monkey work if it helps )

Reply:
Archiving old SB scripts seems the way to go, i agree. - either in a non-compiled folder, or even to another repository?
Test will be updated eventually to use the new version anyway, as we progress with tests for the new features.

Reply:
I suggest we simply delete them from the repo. For historical purposes, there's always the git repo history.

Reply:
So are we deleting this deprecated fx?

Reply:
We can only delete this after we've archived all old sb scripts and updated all the tests, and so there are no dependencies to it, no failing tests, no compile errors. After that is all done: sure we will delete it.

Reply:
done: old sb scripts were archived, above function was deleted.